migrate/20231014_arm64.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_enum(:arch, %w[x64 arm64])
+
+    alter_table(:vm_host) do
+      add_column :arch, :arch
+    end
+
+    alter_table(:vm) do
+      add_column :arch, :arch, default: "x64", null: false
```

Comment:
When discussing product lines, if I recall correctly, we considered incorporating architecture information into the product family as well. Will we have `standard-2` for `amd64` and `standard-2` for `arm64`, or will we introduce a new product family for `arm` separate from `standard`?

Reply:
Good point...and I have no idea, I guess I was kinda thinking it had to be part of the model number, but haven't reconsidered things lately.

I suppose in principle we could try to encode it into the model and avoid a separate column, but somehow this strikes me as too clever by half: the machine architecture is a fairly intrusive concept, so it'd be nice to have it in a column to do filters and joins.

Reply:
(I think we can somewhat harmlessly obsolete this column if it turns out better to do it that way, but as-is this gives us enough power to test out arm64 via REPL)

Reply:
It might be nice to have it in a column. In the early days, we had a single vm_size column (`standard-2`), which we later divided into two: family (`standard`) and cores (`2`).

We might maintain this separation but introduce a differentiator to the VM size value.

Here's an example:
- `family: standard`, `cores: 2`, `arch: x64` -> `standard-4`
- `family: standard`, `cores: 2`, `arch: arm64` -> `standard-4-arm`

Reply:
Separate column makes it easier to reference this from other contexts, such as billing. I think it is fine to keep it as is.